src/options.ts

```diff
@@ -57,7 +56,6 @@ export interface Options {
   endpoint?: string;
   serviceName: string;
   accessToken: string;
-  maxAttrLength: number;
```

Comment:
I'd rather remove that and leave only the most essential config options for programmatic APIs because:

1. Having those provides very little value but has overhead;
2. There's a lot of those missing options and relationships between them - example of `OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT` and `OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT` is one of them;
3. In an extreme vase it can be mimicked bit just setting `process.env` properties.

In conclusion: If we should allow this, why shouldn't we implement the rest of the limits + the general limits?

Reply:
OK fine with removing and allowing only env var based config for limits